h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -68,23 +65,15 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
     # psuedo-class to a clicked element.
     @toolbar.on('mouseup', 'a', (event) -> $(event.target).blur())
 
-  show: -> this.toolbar.removeClass('annotator-hide')
-
-  hide: -> this.toolbar.addClass('annotator-hide')
-
-  onSetTool: (name) ->
```

Comment:
`onSetTool` doesn't have to do with this PR and is just dead code. Feel free to just nuke it on master and then rebase this.